03_configure_undercloud.sh

```diff
@@ -8,7 +8,7 @@ export OS_CLOUD=standalone
 
 openstack endpoint list
 openstack flavor create --ram 1024 --disk 10 --vcpu 2 --public tiny
-openstack flavor create --ram 10240 --disk 20 --vcpu 2 --public m1.medium
+openstack flavor create --ram 8192 --disk 20 --vcpu 2 --public m1.medium
 openstack network create --external --provider-physical-network datacentre --provider-network-type flat public
```

Comment:
The commit message says 4096 but the change is 8192?

Reply:
Oops.  I guess I ammended the commit message but forgot to include the actual change.

I started with 8192, as that matches the AWS flavor in use, but that wasn't actually small enough to work in my test env.

A better solution here is making flavor a deployment choice, and i've got an installer patch in progress to add that.  I still think 4096 is probably a kinder default though.

Reply:
4 GB is now the libvirt default - https://github.com/openshift/installer/pull/785/files